tests/parametric/test_otel_span_methods.py

```diff
@@ -330,21 +588,797 @@ def test_otel_set_span_status_ok(self, test_agent, test_library):
+                span.set_attributes({"http.request.method": "GET"})
+                span.end_span()
+        traces = test_agent.wait_for_num_traces(1)
+        trace = find_trace_by_root(traces, otel_span(name="otel_span_name"))
+        assert len(trace) == 1
+
+        root_span = get_span(test_agent)
+
+        assert root_span["name"] == "http.server.request"
+        assert root_span["resource"] == "otel_span_name"
+
+    @missing_feature(context.library == "golang", reason="Not implemented")
+    @missing_feature(context.library == "java", reason="Not implemented")
+    @missing_feature(context.library == "nodejs", reason="Not implemented")
+    @missing_feature(context.library == "dotnet", reason="Not implemented")
+    @missing_feature(context.library == "python", reason="Not implemented")
+    @missing_feature(context.library == "python_http", reason="Not implemented")
+    def test_otel_span_operation_name_http_client(self, test_agent, test_library):
+        """
+            Tests that the operation name will be set to "http.client.request" when:
+            - Span kind is set to Client
+            - http.request.method is set to something (e.g., GET in this example)
+
+            (https://opentelemetry.io/docs/specs/otel/trace/semantic_conventions/http/)
+        """
+        with test_library:
+            with test_library.otel_start_span("otel_span_name", span_kind=SK_CLIENT) as span:
+                span.set_attributes({"http.request.method": "GET"})
+                span.end_span()
+        traces = test_agent.wait_for_num_traces(1)
+        trace = find_trace_by_root(traces, otel_span(name="otel_span_name"))
+        assert len(trace) == 1
+
+        root_span = get_span(test_agent)
+
+        assert root_span["name"] == "http.client.request"
+        assert root_span["resource"] == "otel_span_name"
+
+    @missing_feature(context.library == "golang", reason="Not implemented")
+    @missing_feature(context.library == "java", reason="Not implemented")
+    @missing_feature(context.library == "nodejs", reason="Not implemented")
+    @missing_feature(context.library == "dotnet", reason="Not implemented")
+    @missing_feature(context.library == "python", reason="Not implemented")
+    @missing_feature(context.library == "python_http", reason="Not implemented")
+    def test_otel_span_operation_name_database(self, test_agent, test_library):
+        """
+            Tests that the operation name will be set to `db.system + "." + "query" when:
+            - Span kind is set to Client
+            - db.system is set to some messaging system (e.g., redis in this example)
+
+            (https://opentelemetry.io/docs/specs/semconv/database/database-spans/)
+        """
+        with test_library:
+            with test_library.otel_start_span("otel_span_name", span_kind=SK_CLIENT) as span:
+                span.set_attributes({"db.system": "redis"})
```

Comment:
For all of these, let's just put a capital letter somewhere in them, to make sure it gets put to lowercase. e.g. this can be `"Redis"` and it should yield `"redis.query"`

Reply:
I added a capital to tags that map to the operation name in c029a1dd21878885ac5cf8d5faf3ea1e998765a3